phpBB/language/en/posting.php

```diff
@@ -241,7 +241,7 @@
 	'SMILIES_ARE_ON'			=> 'Smilies are <em>ON</em>',
 	'STICKY_ANNOUNCE_TIME_LIMIT'=> 'Sticky/Announcement time limit',
 	'STICK_TOPIC_FOR'			=> 'Stick topic for',
-	'STICK_TOPIC_FOR_EXPLAIN'	=> 'Enter 0 for a never ending Sticky/Announcement. Please note that this number is relative to the date of the post.',
+	'STICK_TOPIC_FOR_EXPLAIN'	=> 'Enter 0 for a never ending Sticky/Announcement/Global. Please note that this number is relative to the date of the post.',
```

Comment:
You are correct I missed that one. I'm still on the fence on whether the change is necessary as well and appreciate the feedback. I tend to err on the side of caution and over-explaining. I can live with having whomever merges make the decision also. They've probably got more experience with phpBB code development than I.